.gitlab-ci.yml

```diff
@@ -59,17 +65,22 @@ package-production:
     only:
         - tags
 
-# Not executed, parent job definition for Tomcat deployments
+# Not executed, parent job definition for deployments
 .deploy:
     stage: deploy
     image: alpine:3.9.5
     environment:
         name: $ENVIRONMENT_NAME
     script:
-        - apk add --update curl
-        - curl -u $TOMCAT_USER:$TOMCAT_PASSWORD -T "$ARTIFACT_PATH" "http://$TOMCAT_HOST/manager/text/deploy?update=true&path=/eva/webservices/identifiers&version=$DATETIME" | grep "OK - Deployed application"
+        - apk add --update curl openssh-client
+        - curl -u $TOMCAT_USER:$TOMCAT_PASSWORD -T "$ACC_WS_ARTIFACT_PATH" "http://$TOMCAT_HOST/manager/text/deploy?update=true&path=/eva/webservices/identifiers&version=$DATETIME" | grep "OK - Deployed application"
+        # Start ssh-agent and set identity then copy the Jar file and create a symlink
+        - eval "$(ssh-agent)"; echo "$EVA_ETL_PRIVATE_KEY" > identity_id_rsa; chmod 600 identity_id_rsa; ssh-add identity_id_rsa
+        - export ACC_CLUST_ARTIFACT_NAME=eva-accession-clustering-${ENVIRONMENT_NAME}-${DATETIME}.jar
+        - if [[ ! -z "$CLUSTERING_PATH" ]]; then scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null $ACC_CLUST_ARTIFACT_PATH ${CLUSTER_USER}@${CLUSTER_LOGIN_NODE}:${CLUSTERING_PATH}/${ACC_CLUST_ARTIFACT_NAME} ; fi
+        - if [[ ! -z "$CLUSTERING_PATH" ]]; then ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${CLUSTER_USER}@${CLUSTER_LOGIN_NODE} "ln -fns ${CLUSTERING_PATH}/${ACC_CLUST_ARTIFACT_NAME} ${CLUSTERING_PATH}/eva-accession-clustering.jar"; fi
```

Comment:
Do you think its necessary to have the `if` statement? I think we can count on the path being valid at this stage

Reply:
I think checking the path is warranted in a corner case: NFS screws up (happens sometimes) between the scp and ssh and an invalid symlink is left behind....

Reply:
Ok, I just checked and **ln** still acts sensibly with -f switch and checks for non-existent files....

Reply:
The purpose of the if statement was simply to avoid deploying in `deploy-internal` and `deploy-production-fallback`. Now that the WS and jar have separate deployments the statements are not useful and I've removed them.